docs/reference/configuration/keywords.rst

```diff
@@ -1748,6 +1748,25 @@ bjobs.
                 QUEUE_OPTION TORQUE NUM_NODES 3
                 QUEUE_OPTION TORQUE NUM_CPUS_PER_NODE 8
 
+.. _torque_memory_per_job:
+.. topic:: MEMORY_PER_JOB
+
+
+        You may indicate to the compute cluster how much memory your job will need to
```

Comment:
`You can specify the amount of memory you will need for running the job on the compute cluster.`  ?

Reply:
Yes, fixed.